### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by majianpeng**

### CODE REVIEW

File: src/osd/ReplicatedPG.cc [hunk 1]
```diff
@@ -422,6 +422,7 @@ void ReplicatedPG::wait_for_unreadable_object(
 void ReplicatedPG::wait_for_all_missing(OpRequestRef op)
 {
   waiting_for_all_missing.push_back(op);
+  op->mark_delayed("waiting for alll missing");
```
Comment (liewegas) [inline src/osd/ReplicatedPG.cc]: s/alll/all/

File: src/osd/ReplicatedPG.cc [hunk 2]
```diff
@@ -1267,6 +1268,7 @@ void ReplicatedPG::do_request(
 	     << " flushes_in_progress pending "
 	     << "waiting for active on " << op << dendl;
     waiting_for_active.push_back(op);
+    op->mark_delayed("waiting for flushes end");
```
Comment (liewegas) [inline src/osd/ReplicatedPG.cc]: waiting for flush

**Comment by liewegas** [comment]
otherwise :+1:

**Comment by majianpeng** [comment]
updated.  Thanks!

**Pull request merged**